### DISCUSSION TITLE
Points and Progress!!!

### DISCUSSION THREAD

**Discussion by ReadySetGooo** [ReadySetGooo/NoNameYet#14] [General]
Write what you're working on, whether that be progress on this wiki or progress inspired by the wiki!

**Comment by ReadySetGooo** [comment]
Right now I'm archiving tiktok videos!

**Reply by Ohrldu** [reply]
Awesome!

**Comment by ReadySetGooo** [comment]
I just added some explanations to confusing 17 goals!

**Reply by Ohrldu** [reply]
Fantastic!

**Comment by Ohrldu** [comment]
I added some resources to the add resources post!

**Reply by ReadySetGooo** [reply]
Yay! Thank you so much!!!

**Comment by ReadySetGooo** [comment]
Today I made edits to follow up with a networking contact!
